docs/workflows/build-pages-with-storybook.md

```diff
@@ -108,7 +108,7 @@ LoggedOut.parameters: {
 
 #### Mocking imports
 
-It is also possible to mock imports directly, similar to Jest, using webpack’s aliasing. This is useful if your component makes network requests directly.
+It is also possible to mock imports directly, similar to Jest, using webpack’s aliasing. This is useful if your component makes network requests directly via a library, such as [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch).
```

Comment:
@tmeasday we probably can improve this a bit. We could go with:

"It is also possible to mock imports directly, similar to Jest, using webpack´s aliasing. This is extremely useful if your component makes network requests directly with third-party libraries

We're going to use [isomorphic-fetch](ttps://www.npmjs.com/package/isomorphic-fetch) as a example"